### PR TITLE
[10.0] account_journal_lock_date

### PR DESCRIPTION
**Pull request by sbidoul**
Accounting teams often need to lock sale/purchase journal while leaving misc ops journals open. 

This modules allows this by providing a per-journal lock date.

### CODE REVIEW

**Review by adrienpeiffer — APPROVED** [review]
:+1: (Functional test and code review)

**Review by espo-tony — APPROVED** [review]
:+1:

**Comment by pedrobaeza** [comment]
@sbidoul It would be interesting to include a renaming of the module in OpenUpgrade and handle the values.

File: account_journal_lock_date/views/account_journal.xml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    </record>
+
+
+    <record model="ir.ui.view" id="account_journal_tree_view">
+        <field name="name">account.journal.tree (in account_journal_lock_date)</field>
+        <field name="model">account.journal</field>
+        <field name="inherit_id" ref="account.view_account_journal_tree"/>
+        <field name="arch" type="xml">
+            <field name="type" position="after">
+                <field name="journal_lock_date"/>
```
Comment (rgarnau) [inline account_journal_lock_date/views/account_journal.xml:26]: This tree view is not working, instead you see the view_account_bank_journal_tree.

Comment (sbidoul) [reply account_journal_lock_date/views/account_journal.xml:26]: This is a problem with Odoo view priorities, I proposed a PR in https://github.com/odoo/odoo/pull/17274

**Pull request merged**